src/langs/json/en.json

```diff
@@ -93,7 +93,7 @@
     "confirm": "Confirm remove user",
     "confirmMsg": "Proceeding will remove {name} from {title}. Do you wish to continue?",
     "addCollaborators": "Add Collaborators",
-    "invite": "Put in the email of an existing DocumentCloud user below. If they don't have an account, have them register <a target=\"_blank\" href=\"https://accounts.muckrock.com/accounts/signup/?intent=documentcloud\" >here for free</a>, and then ask them to log in to DocumentCloud at least once.",
+    "invite": "Put in the email of an existing DocumentCloud user below. If they don't have an account, have them register <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://accounts.muckrock.com/accounts/signup/?intent=documentcloud\">here for free</a>, and then ask them to log in to DocumentCloud at least once.",
```

Comment:
Why the `noopener noreferrer` here?  This is a first party link.

Reply:
Really just my habit here. I always put those on `target=_blank` links. I think browsers now implicitly add it, so it's possible we don't need it.